tests/CMakeLists.txt

```diff
@@ -1,11 +1,12 @@
 cmake_minimum_required(VERSION 3.0.2)
 
 project(GSLTests CXX)
+enable_testing()
 
 include(FindPkgConfig)
 include(ExternalProject)
 
-find_package(Git REQUIRED QUIET)
+find_package(Git QUIET)
```

Comment:
Add a comment explaining why the REQUIRED isn't necessary.

Reply:
In fact, there are conditions under which Git would be needed. One of globally installed GTest or Git command is required. I have an idea to express this clause more precisely and move call to find_package() right below [GTest availability check][1] or into the CMakeLists.txt.in template that [actually uses Git][2].

 [1]: https://github.com/microsoft/GSL/pull/963/files#diff-4461c617ceae0c7e0206622aacdf555aedd62eb129c2efb74c84fa1567bcbe0dR15
 [2]: https://github.com/microsoft/GSL/blob/a47352cb2a05cc2684877d8a55077d092f20574b/tests/CMakeLists.txt.in#L6-L7